lib/membrane_vpx/encoder/vp8_encoder.ex

```diff
@@ -1,6 +1,15 @@
 defmodule Membrane.VP8.Encoder do
   @moduledoc """
-  Element that encodes a VP8 stream
+  Element that encodes a VP8 stream.
+
+  This element can receive a `Membrane.VPx.ForceKeyframeEvent` on it's `:output` pad to force the
```

Comment:
I believe you have forgotten to change the docs here in in `vp9_encoder.ex`:
```suggestion
  This element can receive a `Membrane.KeyframeRequestEvent` on it's `:output` pad to force the
```